resources/views/auth/subscription_checkout.php

```diff
@@ -144,21 +144,25 @@
         });
 
     <?php elseif (!empty($plan->stripe_id)): ?>
-        $("#zype-card-date").mask("99/99");
-        $("#zype-card-number").mask("9999 9999 9999 9999");
+        $(".zype-card-date").mask("99/99");
+        $(".zype-card-number").mask("9999 9999 9999 9999");
 
         Stripe.setPublishableKey('<?php echo $stripe_pk ?>');
         $(".zype-checkout-button").prop('disabled', false);
 
         $(".zype-checkout-button").click(function(e) {
             e.preventDefault();
+            var stripeForm = $(this).closest('#payment-form').children('#stripe-form');
```

Comment:
Why do we have to look it up like this, shouldn't just $('#stripe-form') be enough?

Reply:
For the same reason as the one above